Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -504,7 +504,7 @@ public class PBXProjGenerator {
     func getSingleGroup(path: Path, mergingChildren children: [String], depth: Int = 0) -> PBXGroup {
         let group: PBXGroup
         if let cachedGroup = groupsByPath[path] {
-            cachedGroup.children += children
+            cachedGroup.children = Array(Set(cachedGroup.children + children))
```

Comment:
This means we lose ordering of files

Reply:
And if we just sort this we lose alphabetical ordering of files

Reply:
By the way, I changed things up so that all files and groups are now sorted at the end, so we don't have to worry about this sort of thing anymore